packages/core/src/batch/BatchDrawCall.js

```diff
@@ -9,12 +9,17 @@ export class BatchDrawCall
 {
     constructor()
     {
-        this.textures = [];
-        this.ids = [];
+        this.texArray = null;
         this.blend = 0;
-        this.textureCount = 0;
+        this.type = 4;
```

Comment:
I know this is a hold over, but what is `4`?

Reply:
DRAW_MODES.TRIANGLES

Reply:
I understand not wanting to import to save space, could you inline `// DRAW_MODES.TRIANGLES` here?